.github/workflows/publish.yml

```diff
@@ -0,0 +1,39 @@
+name: publish
+
+on:
+  push:
+    paths:
+      - "src/**"
+      - ".github/workflows/publish.yml"
+    branches:
+      - main
+  pull_request:
+    paths:
+      - "src/**"
+      - ".github/workflows/publish.yml"
+
+jobs:
+  build:
+    strategy:
+      matrix:
+        machine: [windows-latest, macos-latest, ubuntu-20.04]
+        architecture: [x64, arm64]
+        include:
+          - machine: windows-latest
+            platform: win
+          - machine: macos-latest
+            platform: osx
+          - machine: ubuntu-20.04
+            platform: linux
+    runs-on: ${{ matrix.machine }}
+    timeout-minutes: 5
+    steps:
+      - name: checkout
+        uses: actions/checkout@v1
+      - name: install linux prerequisites
```

Comment:
https://learn.microsoft.com/en-us/dotnet/core/deploying/native-aot/#prerequisites

Reply:
this needs to be different on windows runners though?

Reply:
Yea I tried googling on the way to install these and wasn't finding anything
 >Desktop development with C++ workload.
My thought was can I just use what's done in the example docker file to set it up https://github.com/dotnet/samples/blob/main/core/nativeaot/HelloWorld/Dockerfile.windowsservercore-x64#L5-L6

Reply:
the Windows runner might already have this preinstalled

Reply:
(you need an extra empty line after a block quote to exit block quote mode)

Reply:
Hey you're right. Linux doesn't need the prerequisites installed either 🎉